restx-jongo/src/test/java/restx/specs/mongo/GivenJongoCollectionTest.java

```diff
@@ -75,8 +86,8 @@ public void should_generate_yaml() throws Exception {
 
         String actual = testCase.toString();
         System.out.println("actual = " + actual);
-        assertThat(actual).isEqualTo(Resources.toString(
+        assertThat(replaceNewLine(actual)).isEqualTo(replaceNewLine(Resources.toString(
```

Comment:
Is it really worth replacing new lines on *both* sides here ? (sounds like useless as it is, but I may be missing something)